docs/data/tree-view-component-api-pages.ts

```diff
@@ -1,8 +1,20 @@
 import type { MuiPage } from '@mui/monorepo/docs/src/MuiPage';
 
 export default [
```

Comment:
It's better to do this:
```suggestion
const apiPages: MuiPage[] = [
```
instead of `as MuiPage[]` which doesn't do typechecking.
Same for other api pages files